lightning-tag/detailed-reference.md

```diff
@@ -43,6 +43,7 @@ onProfileReady | Callback function called once Lotame targeting returns. Discuss
 audienceLocalStorage | Determines whether to write audiences to the browser's `localStorage` and what the name of the key is. Discussed further below in the Local Storage section | `boolean` _**-OR-**_ `string` | NO | `false`
 onTagReady | Callback function called once Lotame Lightning Tag object is loaded and ready to be used. | `function (namespace)` | NO | `{}`
 autoRun | Determines whether Lotame Lightning Tag runs on load. If set to false, your website needs to explicitly call the `page()` method listed on this page to send data and retrieve targeting audiences. | `boolean` | NO | `true`
+bcpPrefix | Overrides the default bcp prefix. Setting it to `cn` will enable data collection in China. | `string` | NO | `bcp`
```

Comment:
will cn work for non-china?  is there a cost to us for people using cn vs bcp ?  

just thinking that reading this I would default to always wanting CN since then I'd get regular + china ; but knowing how the system works i think everything would then silod to the china stack?  and not pick up users data from other sites using bcp ?  

should we make that explanation more clear?  or make it more of a BCP || China Collection to indicate/imply that you can get China OR the rest of the world ?  (also not strictly true but probably closer from a client's perspective?)

Reply:
I agree that we should make this clearer.